File: _etc/C#/16917 양념 반 후라이드 반.cs
using System;
using static System.Console;

class Program
{
    static void Main() {
        string[] s = ReadLine().Split(' ');
        int a = int.Parse(s[0]); int b = int.Parse(s[1]); int c = int.Parse(s[2]); 
        int x = int.Parse(s[3]); int y = int.Parse(s[4]);
        
        int min = Math.Min(x,y);
        int price = 0;
        if(a+b > 2*c) {
            if(x >= y) price = c*2*min + (x - min) * a;
            else price = c*2*min + (y - min) * b;
        }
        else price = a*x + b*y;
        
        price = Math.Min(price, c*2*Math.Max(x,y));
        WriteLine(price);
    }
}

File: _etc/Java/16917 양념 반 후라이드 반.java
import java.util.*;

public class Main {
    public static void main(String args[]) {
      Scanner sc = new Scanner(System.in);
      int a = sc.nextInt(); int b = sc.nextInt(); int c = sc.nextInt();
      int x = sc.nextInt(); int y = sc.nextInt(); int price = 0;
      if(a + b > 2*c) {
          if(x >= y) price = c*2*y + (x-y)*a;
          else price = c*2*x + (y-x)*b;
      }
      else price = a*x + b*y;
      price = Math.min(price, Math.max(x,y) * c * 2);
      System.out.println(price);
    }
}
